.gitattributes

```
@@ -0,0 +1,2 @@
+tests/generated linguist-generated=true
+benches/generated linguist-generated=true
```

```suggestion
/tests/generated/** linguist-generated=true
/benches/generated/** linguist-generated=true
```
I think you can't just target a directory, you need to specify any file in it

---

Good catch, fixed!